mava/utils/logger_tools.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, cfg: Dict) -> None:
         self.should_log = bool(
             cfg["logger"]["use_json"] or cfg["logger"]["use_tf"] or cfg["logger"]["use_neptune"]
         )
+        self.num_eval_episodes = cfg["arch"]["num_eval_episodes"]
```

Comment:
Why is this here? Doesn't seem like we use it

Reply:
Because we need it to calculate the win rate in logger.py: `win_rate = (n_won_episodes / logger.num_eval_episodes) * 100`